itest/assertions.go

```diff
@@ -69,9 +70,12 @@ func AssetAnchorCheck(txid, blockHash chainhash.Hash) AssetCheck {
 			return fmt.Errorf("asset is missing chain anchor field")
 		}
 
-		if a.ChainAnchor.AnchorTxid != txid.String() {
+		anchorTxid :=
+			strings.Split(a.ChainAnchor.AnchorOutpoint, ":")[0]
```

Comment:
Since the txid info was duplicate I now retrieve it by splitting the outpoint string

Reply:
I think you should use `wire.NewOutPointFromString` here.

Reply:
I think you should use `wire.NewOutPointFromString` here.

Reply:
Good call, is the hash reversed in the raw outpoint string?

Reply:
Yes, the outpoint string consists of the `txid` and the `vout` index. The `txid` in the outpoint string is a hex encoding of the byte reversed hash.